docs/tutorials/gradients.ipynb

```diff
@@ -503,19 +519,19 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 0,
       "metadata": {
-        "colab": {},
         "colab_type": "code",
-        "id": "19FKgu0ANT_7"
+        "id": "19FKgu0ANT_7",
+        "colab": {}
       },
-      "outputs": [],
       "source": [
         "test_value = 0.\n",
         "\n",
         "print('Finite difference:', my_grad(pauli_z, test_value))\n",
         "print('Sin formula:      ', -np.pi * np.sin(np.pi * test_value))"
-      ]
+      ],
+      "execution_count": null,
+      "outputs": []
```

Comment:
Once you get the notebook working, don't forget to restart it and clear the outputs.

Reply:
This is still how it looks after going to "Runtime > Restart runtime" and then doing "Edit > Clear all outputs" before downloading